CHANGES.rst

```diff
@@ -72,6 +72,8 @@ Bugs fixed
   Patch by Colin Marquardt.
 * #11598: Do not use query components in URLs for assets in EPUB rendering.
   Patch by David Runge.
+* #11917: Fixes some corner-cases involving inherited class members on python 3.9
+  and earlier. Patch by Janet Carson.
```

Comment:
```suggestion
* #11917: Fix rendering of annotated inherited members for Python 3.9.
  Patch by Janet Carson.
```

We do not support 3.8 and below so there is no need to mention it.

Reply:
added